build.yaml

```diff
@@ -694,13 +811,124 @@ common_cli:
     # /tmp of the slave when the slave is instantiated.
     . /tmp/s3_staging_docs_clusterhq_com.sh
 
+  get_s3_creds_for_bucket_vagrant_jenkins_boxes: &get_s3_creds_for_bucket_vagrant_jenkins_boxes |
+    # collects the S3 credentials that allows us to upload files to the S3
+    # vagrant-jenkins_boxes for our vagrant boxes
+    # the file with the credentials are deployed by the jenkins master to the
+    # /tmp of the slave when the slave is instantiated.
```

Comment:
Does this happen in this file? Either way can you please add a "see also" pointing to where this happens (esp. if it's in another file or another git repo).